docker/default.conf.template

```diff
@@ -23,6 +23,10 @@ server {
         alias  ${NGINX_ENVSUBST_OUTPUT_DIR}/app.config.json;
     }
 
+    location ~ /assets/app.extensions.json {
```

Comment:
did you test with the run.sh script the mapping is as you expect?

Reply:
still having issues with running the scripts, it seems they need some fixes to be used dev environemnts

Reply:
no, the `run.sh` and `run-content-ce.sh` not working for me